udemy_enroller/udemy_rest.py

```diff
@@ -129,8 +129,7 @@ def login(self, retry=False) -> None:
         if cookie_details is None:
             response = self.udemy_scraper.get(self.LOGIN_URL)
             soup = BeautifulSoup(response.content, "html.parser")
-            csrf_element = soup.find("input", {"name": "csrfmiddlewaretoken"}) or {}
-            csrf_token = csrf_element.get("value")
+            csrf_token = response.cookies["csrftoken"]
```

Comment:
Hey @Asunnya 
Thanks for the contribution.

Could you update to be a .get:
```suggestion
            csrf_token = response.cookies.get("csrftoken")
```